wurst/features/PingHerbs.wurst

```diff
@@ -0,0 +1,55 @@
+package PingHerbs
+import HashMap
+import ChatCommands
+import PlayerExtensions
+import UnitExtensions
+import StringExtensions
+import SimError
+
+public class Pingable

```

Comment:
Can we use a [tuple](https://wurstlang.org/manual.html#tuple-types) for this?

Reply:
Hmm I don't think tuples have the ability to be cast to ints, which would make it hard

Reply:
Yea we can't